ibis/tests/expr/test_table.py

```diff
@@ -546,13 +568,13 @@ def test_order_by_scalar(table, key, expected):
 
 
 @pytest.mark.parametrize(
-    ("key", "exc_type"),
+    "key",
     [
-        ("bogus", com.IbisTypeError),
-        (("bogus", False), com.IbisTypeError),
-        (ibis.desc("bogus"), com.IbisTypeError),
-        (_.bogus, AttributeError),
-        (_.bogus.desc(), AttributeError),
+        "bogus",
+        ("bogus", False),
+        ibis.desc("bogus"),
+        _.bogus,
```

Comment:
FieldsNotFoundError is a subclass of AttributeError, so this is only breaking for the test cases that looked for IbisTypeError.

Reply:
EDIT: now that I made FieldsNotFound NOT a subclass of Attribute error, this is now breaking for all of these test cases.